backend/test/card_game/api_test.clj

```diff
@@ -1,6 +1,7 @@
 (ns card-game.api-test
   (:use expectations
-        card-game.api))
+        card-game.api
+        card-game.persistence))
```

Comment:
apit_test should **_NOT_** know about persistence, at all

Reply:
I also see you don't actually use it... please remove line 🙃

Reply:
My mistake.